insights/parsers/tests/test_parsers_module.py

```diff
@@ -313,3 +346,50 @@ def test_keyword_search():
     assert results[2] == DATA_LIST[3]
     # No data, no results.
     assert len(keyword_search([], role='server')) == 0
+
+    # Search with contains
+    results = keyword_search(DATA_LIST, role__contains='e')
+    assert len(results) == 4
+    assert results[0] == DATA_LIST[0]
+    assert results[1] == DATA_LIST[1]
+    assert results[2] == DATA_LIST[2]
+    assert results[3] == DATA_LIST[3]
+
```

Comment:
@gavin-romig-koch - I'm always happy to use a more compact representation, so I'll use that :-)